xml/issue_template.md

```diff
@@ -0,0 +1,21 @@
+### Type of report
+   - [ ] bug
+   - [ ] feature request
+
+### Source Document
+@@source@@
+
+### Summary
+<!-- A clear and concise description of what the bug/feature request is. -->
```

Comment:
ftr, I think XML-type comments work as comments in Markdown but I am not 100% sure anymore.

Reply:
Yeah, I had the same thought. I think they do. I just need to remember. I'll look it up properly. Consider it a placeholder.